src/doc/en/developer/packaging.rst

```diff
@@ -121,6 +127,8 @@ the following source types:
 
    - the file ``package-version.txt`` is optional;
 
+   - may be associated with a source tree included in the repository;
+
    - installing the package runs the installation script ``spkg-install`` or
```

Comment:
"a source tree included in the repository": 

How about "a source tree in the repository"? 

Is it "included"? Does this mean something I don't get?

Reply:
Thanks. "in the repository" is good enough and is shorter.